dipy/tracking/local/tests/test_local_tracking.py

```diff
@@ -318,11 +330,23 @@ def allclose(x, y):
             raise AssertionError()
 
     # The first path is not possible if 90 degree turns are excluded
-    dg = DeterministicMaximumDirectionGetter.from_pmf(pmf, 80, sphere)
+    dg = DeterministicMaximumDirectionGetter.from_pmf(pmf, 80, sphere,
+                                                      pmf_threshold=0.1)
     streamlines = LocalTracking(dg, tc, seeds, np.eye(4), 1.)
 
     for sl in streamlines:
         npt.assert_(np.allclose(sl, expected[1]))
 
+    # Both path are not possible if 90 degree turns are exclude and
+    # if pmf_threhold is superior to 0.4. Streamlines should stop at
```

Comment:
Small nit: "superior to" is perhaps not the right word. How about "larger than"?